core/src/main/java/be/orbinson/aem/dictionarytranslator/services/DictionaryService.java

```diff
@@ -33,11 +46,41 @@ public interface DictionaryService {
 
     boolean keyExists(Resource dictionaryResource, String language, String key);
 
+    /**
+     * 
+     * @param languageResource
+     * @param key
+     * @return
+     * @deprecated use {@link #getMessages(Resource, String)} instead
+     */
+    @Deprecated
```

Comment:
deprecate or just remove? This is too low-level to support all types of dictionaries.